firedrake/preconditioners/patch.py

```diff
@@ -585,10 +585,13 @@ def coords(dm, p, coordinates):
         bary /= len(closure_of_p)
         return bary
 
-    def sort_entities(self, dm, axis, dir, ndiv):
+    def sort_entities(self, dm, axis, dir, ndiv=-1, divisions=None):
```

Comment:
Use `None` as the out of band value, not `-1`

Reply:
Done.